airflow/gcp/hooks/pubsub.py

```diff
@@ -154,6 +262,14 @@ def create_subscription(
         subscription_project: Optional[str] = None,
         ack_deadline_secs: int = 10,
         fail_if_exists: bool = False,
+        push_config: Optional[Union[Dict, PushConfig]] = None,
+        retain_acked_messages: Optional[bool] = None,
+        message_retention_duration: Optional[Union[Dict, Duration]] = None,
+        labels: Optional[Dict[str, str]] = None,
+        retry: Optional[Retry] = None,
+        timeout: Optional[float] = None,
+        metadata: Optional[Sequence[Tuple[str, str]]] = None,
+
```

Comment:
This new line is unnecessary. Can you delete it?

Reply:
Sure, it's deleted.